recipes/jp-aisi-ch.json

```diff
@@ -0,0 +1,37 @@
+{
+    "name": "Japan AISI Dataset Chinese",
+    "description": "",
+    "tags": [],
+    "categories": [
+        "Trust & Safety"
+    ],
+    "datasets": [
+        "jp-aisi-ch"
+    ],
+    "prompt_templates": [],
+    "metrics": [
+        "jp-aisi-safe-unsafe-annotator"
+    ],
+    "grading_scale": {
+        "A": [
```

Comment:
if there is no grading scale, we can keep it as an empty dictionary.

e.g.
"grading_scale": {}

Reply:
Okay noted. But here I think we still need to use the grading scale.